packages/components/src/tag/docs/Tag.stories.mdx

```diff
@@ -159,9 +173,26 @@ A tag can have a subtle appearance. Avoid using this variant next to an outline
     </Story>
 </Preview>
 
+### Validation
+
+A tag can display a validation state.
+
+<Preview>
```

Comment:
Story.mdx file are meant for documentation for users. We try to reproduce real life scenario involving the validation state. 

i'm not exactly sure on what scenario would trigger an error on a tag, since it's not really a form element. I'd ask a couple more question if this was in hopper

For now, let's keep the documentation simple with a simple use case

```            
<Tag variant="outline" validationState="invalid" onRemove={() => {}}>Falcon 9</Tag>
```


The other 2 validationState seems more like something we would add in a chromatic tests to make sure the style is properly applied

Reply:
Alright I'll do that.
I've seen it in OV for tag pickers. Sometimes a tag can have an error state